erpnext_documentation/www/docs/user/manual/en/regional/india/setup-e-invoicing.md

```diff
@@ -0,0 +1,69 @@
+<!-- add-breadcrumbs -->
+# E Invoicing under GST
```

Comment:
@nextchamp-saqib 

There's no context... please incorporate answers to the following questions in the documentation

Why was e invoicing introduced?
what is GSP? what is IRN? full form? uses? Why would someone wanna cancel IRN?

What is relevance of eway bill in einvoicing?

Reply:
Done! Let me know whats missing now.